src/common/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Button = ({ text, onClick, type, disabled, classnames }) => {
+  return (
+    <button
+      type={type}
+      disabled={disabled}
+      onClick={onClick}
+      className={classnames}
+    >
+      {text}
```

Comment:
The `text` props is not necessary because the content between a tag can be accessed using the `children` reserved props.

Reply:
Feedback noted. Will fix immediately.